src/global_const.js

```diff
@@ -1017,13 +1035,15 @@ var cosmosSkills = {
 module.exports.skillDetails = {
     'victorious-covenant': 'victorious_calamitous_covenant',
     'calamitous-covenant': 'victorious_calamitous_covenant',
-    'shinTenNoInori': 'shinTenNoInori'
+    'shinTenNoInori': 'shinTenNoInori',
+    'slaysnakes_myth': 'slaysnakes_myth',
```

Comment:
- [ ] Add `'slaysnakes_mythII': 'slaysnakes_myth',`
- no need add to skillDetailsDescription

Reply:
Done